code/datums/storage/subtypes/others/pod.dm

```diff
@@ -0,0 +1,50 @@
+/datum/storage/pod
+	max_slots = 14
+	max_total_storage = WEIGHT_CLASS_BULKY * 14
+	allow_big_nesting = TRUE
+	/// If TRUE, we unlock regardless of security level
+	var/always_unlocked = FALSE
+
+/datum/storage/pod/open_storage(mob/to_show)
+	if(isliving(to_show) && SSsecurity_level.get_current_level_as_number() < SEC_LEVEL_RED)
+		to_chat(to_show, span_warning("The storage unit will only unlock during a Red or Delta security alert."))
+		return FALSE
+	return ..()
+
+/datum/storage/pod/New(atom/parent, max_slots, max_specific_storage, max_total_storage)
+	. = ..()
+	// all of these are a type below what actually spawn with
+	// (IE all space suits instead of just the emergency ones)
+	// because an enterprising traitor might be able to hide things,
+	// like their syndicate toolbox or softsuit. may be fun?
+	set_holdable(exception_hold_list = list(
+		/obj/item/clothing/suit/space,
+		/obj/item/pickaxe,
+		/obj/item/storage/toolbox,
+	))
+
+	RegisterSignal(SSsecurity_level, COMSIG_SECURITY_LEVEL_CHANGED, PROC_REF(update_lock))
```

Comment:
Seems an `update_lock(new_level = SSsecurity_level.get_current_level_as_number())` call got lost in copying this over?
```suggestion
	RegisterSignal(SSsecurity_level, COMSIG_SECURITY_LEVEL_CHANGED, PROC_REF(update_lock))
	update_lock(new_level = SSsecurity_level.get_current_level_as_number())
```
Without it I don't think this properly updates on being created.